common.props

```diff
@@ -4,7 +4,7 @@
 
     <MajorProductVersion>4</MajorProductVersion>
     <MinorProductVersion>0</MinorProductVersion>
```

Comment:
nit: We should bump the minor here.

Reply:
Bumped minor version number. New version with this fix will be 4.1.0